docs/perf_analyzer.md

```diff
@@ -402,6 +402,10 @@ The following is the example to provide contents as base64 string with explicit
 }
 ```
 
+Note that for STRING type an element is represented by a 4-byte unsigned integer giving
```

Comment:
Should an example be given for STRING type? Or is there already an example? Is there a difference between a base64 input and a string input?

Reply:
The example is provided right above this line. Base64 is an encoding of a bytes array.